docs/topics/multiplatform/multiplatform-configure-compilations.md

```diff
@@ -36,6 +36,22 @@ kotlin {
 }
 ```
 
+Alternatively, you can use the `compilerOptions` [top-level block](multiplatform-dsl-reference.md#top-level-blocks):
```

Comment:
Maybe worth mentioning here that in both examples only common for all targets compiler options are available.